code generators/de.fhdo.lemma.model_processing.code_generation.container_base/src/main/java/de/fhdo/lemma/model_processing/code_generation/container_base/template/DockerComposeTemplate.xtend

```diff
@@ -0,0 +1,100 @@
+package de.fhdo.lemma.model_processing.code_generation.container_base.template
+
+import java.util.Map
+import de.fhdo.lemma.model_processing.code_generation.container_base.file.docker.Service
+import de.fhdo.lemma.model_processing.code_generation.container_base.file.docker.Network
+
+/**
+ * Template class for generating docker compose files.
+ *
+ * @author [Philip Wizenty](mailto:philip.wizenty@fh-dortmund.de)
+ */
+class DockerComposeTemplate {
+
+    /*
+     * Build the operation node specific docker compose part
+     */
+    static def String buildDockerComposeParts(String name, Service service) '''
+        «"  "»«name.toLowerCase»:
```

Comment:
Err... IMHO you could remove all the `«"  "»` expressions and simple replace them with two spaces _within_ the template as Xtend template strings keep indentation information as you type them.

Reply:
My Eclipse settings automatically removes spaces at the beginning of a line in Xtend's template expressions. 
I tried it without `" "` first too.